sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/resource/status/StatusCountResourceService.groovy

```diff
@@ -64,6 +69,10 @@ class StatusCountResourceService extends ResourceService<StatusCount>{
                 publish(statusCount, Topic.SAMPLE_RECEIVED_COUNT_UPDATE)
                 content.remove(statusCount)
                 break
+            case Status.SAMPLE_QC_PASS:
+                publish(statusCount, Topic.SAMPLE_PASSED_QC_COUNT_UPDATE)
+                content.remove(statusCount)
+                break
```

Comment:
You can merge this case with the failure case I think. But you do not have to, we can also merge it in the view only atm.

Reply:
I would do that in the refactoring of the service. I also thought so but how do I know then which count was updated?